### ISSUE TITLE
Naga error on simple cube

### ISSUE THREAD

**Issue by rewin123** [komadori/bevy_mod_outline#40]
![image](https://github.com/user-attachments/assets/3ebbe5a3-7dcf-438b-bc4c-b13c2a92abd7)

```bash
error: failed to build a valid final module: Type [1] 'VertexOutput' is invalid
  ┌─ C:\Users\rewin\.cargo\git\checkouts\bevy_mod_outline-2e7dc4b49e7473ae\391bf20\src\fragment.wgsl:1:1
  │
1 │ struct VertexOutput {
  │ ^^^^^^^^^^^^^^^^^^^^^ naga::Type [1]
  │
  = Structure types must have at least one member

```

Main branch ([commit](https://github.com/komadori/bevy_mod_outline/commit/391bf20cb9282def126182cf7eba3e91b8bddaf0))

**Comment by komadori** [comment]
This bug causes the stencil fragment shader to be invalid when the outline mode has been set to `OutlineMode::RealVertex`. As a workaround and in any case, you may wish to consider if this is the right setting for you.

Fixed by 664be88b17407383d18b972498ce69094518fd48. I will make a minor release shortly.

**Comment by rewin123** [comment]
![image](https://github.com/user-attachments/assets/72354069-1923-421a-ada4-582ddddc7314)
Working 😄

**Comment by komadori** [comment]
Fix released in 0.8.1.
